src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -403,6 +405,8 @@
     private static final int EXPORTER_FILENAME_INDEX = 1;
     private static final int EXPORTER_EXTENSION_INDEX = 2;
 
+    private static final String PATH_TO_CUSTOM_THEME = "pathToCustomTheme";
```

Comment:
Is this field really required? Can we not simply point `FX_Theme` to the external css file?